pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/MeasurementsKokkos.hpp

```diff
@@ -249,6 +250,52 @@ class Measurements final
         }
     };
 
+    /**
+     * @brief Expected value of an observable.
+     *
+     * @param pauli_words Vector of operators' name strings.
+     * @param target_wires Vector of wires where to apply the operator.
+     * @param coeffs Complex buffer of size |pauli_words|
+     * @return Floating point expected value of the observable.
+     */
+    auto expval(const std::vector<std::string> &pauli_words,
```

Comment:
```suggestion
    auto expval(const std::vector<std::string> &pauli_sentence,
```

I think this will easier to understand